src/Views/Form/Templates/Classic/resources/css/_errors-notices.scss

```diff
@@ -26,6 +33,10 @@
     padding-inline: 0.75rem;
     position: relative;
 
+    &:not(:last-child) {
+        margin-bottom: 0.9rem;
```

Comment:
Do we need both uses of the `margin-bottom` rule, or can one of the two uses be dropped?